scripts/toolchain/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine
+
+LABEL description="Image that contains all utilities necessary to build hopr monorepo, including Rust and Yarn"
+
+# python, perl are used by some nodejs dependencies as an installation requirement
+RUN apk add --no-cache bash build-base ca-certificates jq curl git
+
+WORKDIR /app/hoprnet
```

Comment:
You should use a different workdir here which does not overlap with other Dockerfile's workdir to prevent collisions.